cardano-api/src/Cardano/Api/Utils.hs

```diff
@@ -133,4 +133,3 @@ renderEra (AnyCardanoEra AllegraEra) = "Allegra"
 renderEra (AnyCardanoEra MaryEra)    = "Mary"
 renderEra (AnyCardanoEra AlonzoEra)  = "Alonzo"
 renderEra (AnyCardanoEra BabbageEra) = "Babbage"
-
```

Comment:
My editor really doesn't like multiple blank lines at the end of a file and it serves no purpose. Can we just leave this? I don't want to open an additional PR to remove a single extra blank line

Reply:
There is no need to open a random PR. It should be straight forward to modify the same commit to remove the line removal.

Reply:
This file currently ends with `"\n\n"`. To my mind, this is a formatting mistake that should never have been allowed, but since it has occurred, it should be corrected. I assume it snuck in during a large refactoring PR.
Since it is unwanted noise, I'll remove this change from this PR.